frappe/desk/doctype/notification_log/notification_log.py

```diff
@@ -12,7 +12,10 @@ def after_insert(self):
 		frappe.publish_realtime('notification', after_commit=True, user=self.for_user)
 		set_notifications_as_unseen(self.for_user)
 		if is_email_notifications_enabled_for_type(self.for_user, self.type):
-			send_notification_email(self)
+			try:
+				send_notification_email(self)
+			except frappe.OutgoingEmailError:
+				frappe.log_error(message=frappe.get_traceback(), title=_("Failed to send notification email"))
```

Comment:
Logging without any reference record information does not really help. Can you please include context in error log.